assets/src/edit-story/components/tablist/panel.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import { Icons } from '../../../design-system';
+import {
+  Badge,
+  ButtonText,
+  IconContainer,
+  PanelWrapper,
+  SmallHeadline,
+  TabButton,
+  TabPanel,
+} from './styles';
+
+const Panel = ({
+  children,
+  className,
+  isExpanded,
+  numIssues = 0,
```

Comment:
perhaps something generic like `badgeCount`?

Reply:
I like that way better